cmake/std/PullRequestLinuxDriverTest.py

```diff
@@ -467,19 +496,22 @@ def run():
     re_src_branchname = "master_merge_[0-9]{8}_[0-9]{6}"
     if "master" == arguments.targetBranch:
         if re.match(re_src_branchname, arguments.sourceBranch) is None:
-            sys.exit("""------------------------------------------------------------------------------------------
-NOTICE: Destination branch is trilinos/Trilnos::master
-ERROR : Source branch is NOT trilinos/Trilinos::master_merge_YYYYMMDD_HHMMSS
-      : This violates Trilinos policy, pull requests into the master branch are restricted.
-      : Perhaps you forgot to specify the develop branch as the target in your PR?
-------------------------------------------------------------------------------------------
-""")
+            sys.exit(textwrap.dedent("""\
```

Comment:
I have never understood decent for this type of use. Ehh.

Reply:
I don't follow your comment...  the `\` after the `"""` means no newline and `dedent(<string>)` will de-indent the string by the line with the least amount of leading whitespace... this makes long multiline strings easier to read in code since they don't break the visual indendentation within the code. Mostly it's just for readability.

Reply:
sorry - not decent, dedent. This is a non-issue, just stylistic differences. And me fighting with autocorrect.